boundaries/build/position-data-query-results.json

```diff
@@ -4191,17 +4191,17 @@
         },
         "positionSuperclass": {
           "type": "uri",
-          "value": "http://www.wikidata.org/entity/Q5096847"
+          "value": "http://www.wikidata.org/entity/Q2285706"
```

Comment:
These look to be related to query order changing - is this something we can slate to fix? Not needed for this PR.

Reply:
Ticketed as https://github.com/everypolitician/commons-builder/issues/115.